code/modules/projectiles/projectile.dm

```diff
@@ -199,7 +199,7 @@
 			M.Turn(Angle)
 			transform = M
 
-			var/Pixel_x=round((sin(Angle)+16*sin(Angle)*2), 1)	//Without the 1 it rounds to -1 for cos because byond's functions are shit.
+			var/Pixel_x=round((sin(Angle)+16*sin(Angle)*2), 1)	//round() doesn't round unless you specify 1.

```

Comment:
prefer: "round() is a floor operation when only one argument is supplied, we don't want that here"